static/documentation.html

```diff
@@ -56,31 +56,16 @@ <h5 class="offcanvas-title" id="offcanvasNavbarLabel">Build Your Own EA</h5>
           </div>
           <div class="offcanvas-body">
             <ul class="navbar-nav mr-auto">
-              <li class="nav-item">
-                <a class="nav-link" href="workspace.html?example=full">
-                  Demo
-                  <span class="sr-only">(current)</span>
-                </a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link" href="faq.html">FAQ</a>
-              </li>
               <li class="nav-item">
                 <a class="nav-link" href="documentation.html">Documentation</a>
               </li>
-              <li class="nav-item dropdown">
-                <a
-                  class="nav-link dropdown-toggle"
-                  href="#"
-                  id="navbarExampleDropdown"
-                  role="button"
-                  data-bs-toggle="dropdown"
-                  aria-haspopup="true"
-                  aria-expanded="false"
-                >
-                  Examples
-                </a>
-                <ul
+              <li class="nav-item">
+                <div class="btn-group">
+                  <a type="button" class="btn btn-dark" href="workspace.html?example=empty">Workspace</a>
```

Comment:
Does it look better with a brighter button color that is more in line with the rest of the navbar ? If not we can leave it like that